tests/mirage/crates/get-by-id-test.js

```diff
@@ -223,4 +223,47 @@ module('Mirage | GET /api/v1/crates/:id', function (hooks) {
       versions: null,
     });
   });
+  test('includes default_version', async function (assert) {
+    let crate = this.server.create('crate', { name: 'rand' });
+    this.server.create('version', { crate, num: '1.0.0' });
+    this.server.create('version', { crate, num: '1.1.0' });
+    this.server.create('version', { crate, num: '1.2.0' });
+
+    let req = await fetch('/api/v1/crates/rand');
+    let expected = await req.json();
+
+    let response = await fetch('/api/v1/crates/rand?include=default_version');
+    assert.strictEqual(response.status, 200);
+
+    let responsePayload = await response.json();
+    let default_version = expected.versions.find(it => it.num === responsePayload.crate.default_version);
+    assert.deepEqual(responsePayload, {
+      ...expected,
+      crate: {
+        ...expected.crate,
+        categories: null,
+        keywords: null,
+        max_version: '0.0.0',
+        newest_version: '0.0.0',
+        max_stable_version: null,
+        versions: null,
```

Comment:
This would require a corresponding change if https://github.com/rust-lang/crates.io/pull/10288#discussion_r1900195709 changes.